### PR TITLE
BXMSPROD-8: refactor to build without jboss-ip-bom

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

**Comment by mbiarnes** [comment]
https://github.com/kiegroup/kie-soup/pull/95
https://github.com/kiegroup/appformer/pull/718
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/973
https://github.com/kiegroup/droolsjbpm-knowledge/pull/391
https://github.com/kiegroup/drools/pull/2396
https://github.com/kiegroup/optaplanner/pull/514
https://github.com/kiegroup/jbpm/pull/1503
https://github.com/kiegroup/kie-jpmml-integration/pull/5
https://github.com/kiegroup/droolsjbpm-integration/pull/1849
https://github.com/kiegroup/droolsjbpm-tools/pull/95
https://github.com/kiegroup/kie-uberfire-extensions/pull/85
https://github.com/kiegroup/kie-wb-playground/pull/76
https://github.com/kiegroup/kie-wb-common/pull/2710
https://github.com/kiegroup/drools-wb/pull/1169
https://github.com/kiegroup/optaplanner-wb/pull/339
https://github.com/kiegroup/jbpm-designer/pull/854
https://github.com/kiegroup/jbpm-wb/pull/1364
https://github.com/kiegroup/kie-docs/pull/1645
https://github.com/kiegroup/kie-wb-distributions/pull/940
https://github.com/kiegroup/optaweb-employee-rostering/pull/250

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
@yurloc @ge0ffrey 
Locally it works!
If you look at [jboss.org origin-repository](https://origin-repository.jboss.org/nexus/content/groups/public/org/optaplanner/optaplanner-examples/7.24.0-SNAPSHOT/) you see optaplanner-examples-7.24.0-20190617.102554-24.jar but in [jboss.org repository](https://repository.jboss.org/nexus/content/groups/public/org/optaplanner/optaplanner-examples/7.24.0-SNAPSHOT/) you don't see it.
We shoudln't work with the origin-repositories.

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by yurloc** [comment]
@mbiarnes Great! Please also make it work in Jenkins otherwise we can have a broken build on master after merging, right?

How is the origin-repository related to the build failure? I don't see it mentioned either in Jenkins console or in any optaplanner pom.xml.

It seems that the enforcer plugin error is caused by corrupted nexus repository. The [maven-metadata.xml](https://repository.jboss.org/nexus/content/groups/public/org/optaplanner/optaplanner-persistence-jpa/7.24.0-SNAPSHOT/maven-metadata.xml) says the latest snapshot version is `7.24.0-20190617.085242-23` but no artifact with that version is installed [there](https://repository.jboss.org/nexus/content/groups/public/org/optaplanner/optaplanner-persistence-jpa/7.24.0-SNAPSHOT/).

Can you explain it?

**Comment by mbiarnes** [comment]
Jenkins retest this please

File: optaplanner-docs/pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
   </parent>
 
   <artifactId>optaplanner-docs</artifactId>
-  <packaging>jar</packaging>
+  <packaging>pom</packaging>
```
Comment (ge0ffrey) [inline optaplanner-docs/pom.xml]: Are you sure this doesn't cause trouble for optaplanner-distribution?
Presuming that target/*.zip file still contains the docs, this PR is fine.

**Pull request merged**